Functions/xASL_stat_GetROIstatistics.m

```diff
@@ -177,22 +176,30 @@
 end
 
 %% Define number of sessions to use
-nSessions = 0;
-NoSessions = 0;
-SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*_ASL_' num2str(nSessions+1) '\.nii'], 'FPList', [0 Inf]);
-
-while ~isempty(SessionList)
-       nSessions = nSessions+1;
-       SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*_ASL_' num2str(nSessions+1) '\.nii'], 'FPList', [0 Inf]);
-end
+SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*_ASL_\d*\.nii'], 'FPList', [0 Inf]);

```

Comment:
So if you want to exclude `_masked` you could try something like:

`SessionList = xASL_adm_GetFileList(x.D.PopDir,['^' x.S.InputDataStr '.*(?<!_masked)_ASL_\d*\.nii'], 'FPList', [0 Inf]);`

Per this explanation, check if it should be the lookahead or negative lookahead:

(?! ) is a lookahead (do not match the symbols in the space here that are to the RIGHT)
(?<! ) is a negative lookahead (do not match the symbols in the space here that are to the LEFT)

Reply:
Can't seem to get it to work